apostrophecms/a3-docs
# Pro 3109 heroku tut

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*

This PR adds a new recipe for hosting an apostrophe site on Heroku. It is a port of the a2 recipe with updates. Closes PRO-3109.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Build the documentation
2. Navigate to `cookbook/deploying-to-heroku.html`
3. 
## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
I'm not sure how much of the section after "Victory!" is needed and/or accurate. I'm not very sure about CORS
